-------------------------------------------------------------------------------
Test set: com.jerae.jsecurity.NewFeatureTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.133 s <<< FAILURE! -- in com.jerae.jsecurity.NewFeatureTest
com.jerae.jsecurity.NewFeatureTest.testWarnCommand -- Time elapsed: 0.026 s <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <testPlayer has been warned by sender for test reason.> but was: <testPlayer has been warned by Console for test reason.>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1141)
	at com.jerae.jsecurity.NewFeatureTest.testWarnCommand(NewFeatureTest.java:107)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
